### PR TITLE
Add elm hello world

### PR DESCRIPTION
**Pull request by mjhoy**

### CODE REVIEW

**Comment by mjhoy** [comment]
@camenpihor got it working, i forked the `setup-elm` action and added this commit: https://github.com/mjhoy/setup-elm/commit/c4aeaf781b80b172eec41d15bff23cc0fbdcd74a

i also hard-coded the test here that depended on the date: e89433d we should probs rethink how that works eventually

File: backend/tests/test_coviz.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ def test_read_confirmed_time_series():
     non_date_columns = ["Province/State", "Country/Region", "Lat", "Long"]
     date_columns = list(
         pd.date_range(
-            start="2020-01-22", end=datetime.date.today() - datetime.timedelta(days=2)
+            start="2020-01-22", end="2020-04-03"
```
Comment (camenpihor) [inline backend/tests/test_coviz.py:11]: hmm this should work if you `git submodule update` I thought that i added that in the error message

**Comment by camenpihor** [comment]
Nice on the commit to setup-elm!

**Comment by mjhoy** [comment]
> Happy to approve, but I think we don't need to hardcode in the date on the test (also happy to hardcode it for now if you want)

the commit that the covid is checked out at is also hard-coded, though. so i think we should bump the test when covid is bumped (it will be part of the same commit), and also perhaps we could figure out a different way of testing or bumping in the future

**Pull request merged**